### ISSUE TITLE
[Bug]: Playwright Test for VSCode extension v.1.1.1 - Run button does not appear next to tests

### ISSUE THREAD

**Issue by mjeruzal427** [microsoft/playwright#30819]
### Version

1.44.0

### Steps to reproduce

1. Install / update to the latest Playwright Test for VSCode extension (version 1.1.1)
2. Open any spec.ts file

### Expected behavior

The Run Test button should appear next to the lines with tests like this:
![image](https://github.com/microsoft/playwright/assets/59467031/c7e9a000-3cac-4182-90a7-f1f27eaca7d9)

### Actual behavior

The button is not there anymore.

### Additional context

- Installing the previous version of the extension (1.0.22) fixes this issue.
- I am using the latest version of Visual Studio Code (1.89.1)

### Environment

```shell
System:
    OS: Windows 11 10.0.22631
    CPU: (8) x64 Intel(R) Core(TM) i5-8365U CPU @ 1.60GHz
    Memory: 16.11 GB / 31.82 GB
  Binaries:
    Node: 18.16.1 - C:\Program Files\nodejs\node.EXE
    npm: 9.7.2 - C:\Program Files\nodejs\npm.CMD
  IDEs:
    VSCode: 1.89.1 - C:\Users\mje\AppData\Local\Programs\Microsoft VS Code\bin\code.CMD
  Languages:
    Bash: 5.2.15 - C:\Program Files\Git\usr\bin\bash.EXE
  npmPackages:
    @playwright/test: ^1.44.0 => 1.44.0
```

**Comment by mxschmitt** [comment]
Does this also happen with a [newly created project](https://playwright.dev/docs/intro#installing-playwright)?

Ideally we have something which we can run locally in order to reproduce this bug, thanks for your understanding!

**Comment by mjeruzal427** [comment]
@mxschmitt Yes, it does. I've initialized a brand new playwright project by using the command:
```bash
npm init playwright@latest
```
The project was not modified by me at any step.

With the **latest** (1.1.1) extension available "Run Test" buttons do not appear for `example.spec.ts`:
![image](https://github.com/microsoft/playwright/assets/59467031/4e895c78-4b97-4153-b586-dec46ce6f068)

After rolling back to **previous** version of extension (1.0.22) the buttons are available:
![image](https://github.com/microsoft/playwright/assets/59467031/92fa4280-b452-44c9-9c1b-4bf735dd0ef8)

**Comment by dsayerm** [comment]
We noticed the same thing when we upgraded. On further investigation we found that we need to set the Projects we wish to have included in the test runner:
![image](https://github.com/microsoft/playwright/assets/86229263/158cd84e-5337-4049-85e1-30fa6d66a8a2)

This configuration was stored then on /.vscode/settings.json.

**Comment by Tallyb** [comment]
hey @mxschmitt 👋 
Experiencing something similar. I suspect it happens when you have mutliple projects on your config pointing to different folders (e.g. 
```
{
   name: 'foo'
  testDir: '.projects/foo
},
{
   name: 'bar'
  testDir: '.projects/bar
},
```
only foo will be picked up. you will also that in the tests explorer - only first project. 

(PS only tested extension v1.1.0 but PW is on 1.43.1. It will take couple more days before we will upgrade)

**Comment by jtgober** [comment]
I had a similar issue and found this post (also have multiple projects) For me it was also skipping my global setup for whatever reason. However using what @dsayerm posted with the projects toggles seems to have fixed it for me at the moment. Is that a new feature? I don't recall seeing that one before

**Comment by yury-s** [comment]
> With the latest (1.1.1) extension available "Run Test" buttons do not appear for example.spec.ts:

@mjeruzal427 can you share screenshot of the Playwright Test extension sidebar in that case?

**Comment by mjeruzal427** [comment]
@yury-s list of tests is stuck on "Loading..." forever. Also, I don't have any options available under "Projects" section as @dsayerm and @jtgober suggested.
![image](https://github.com/microsoft/playwright/assets/59467031/57ce6929-e9bf-49d7-85b1-9c7b2997d3d8)

**Comment by vladkrasn** [comment]
Yeah this seems to be tied to the new Projects tab in the extenstion. Would be nice if Play button appeared even if the project is disabled

**Comment by yury-s** [comment]
@mjeruzal427 Could you please share console output from Developer Tools console, you can open it from command menu

![image](https://github.com/microsoft/playwright/assets/9798949/feee3e6c-2fd8-4956-a958-1444f3c691df)

And also Extension Host logs. From the command menu (Command+Shift+P on Mac): Developer: Show Logs... >  Extension Host, and then copy the logs:

<img width="1051" alt="image" src="https://github.com/microsoft/playwright/assets/9798949/25a0b619-479b-4b99-a9ed-d4226047de6f">

**Comment by mjeruzal427** [comment]
@yury-s, @pavelfeldman The problem has been resolved after I installed the latest version (1.1.5) of the extension. This issue can be closed now. Thank you.
